src/testers.jl

```diff
@@ -256,6 +256,7 @@ end
 - `x`: input at which to evaluate `f` (should generally be set to an arbitary point in the domain).
 - `x̄`: currently accumulated adjoint (should generally be set randomly).
 
+Non-differentiable arguments, such as indices, should have `x̄` set as `nothing`.
```

Comment:
@nickrobinson251 I've added this line as well as an abridged version of Lyndon's explanation. Would you mind taking another look?

Reply:
bump

Reply:
one more bump